testing/test_warnings.py

```diff
@@ -39,9 +39,9 @@ def foo():
 
 
 @pytest.mark.filterwarnings("default")
-def test_normal_flow(testdir, pyfile_with_warnings):
+def test_normal_flow(pytester: Pytester, pyfile_with_warnings):
```

Comment:
```suggestion
def test_normal_flow(pytester: Pytester, pyfile_with_warnings) -> None:
```

Reply:
Understood. This is to be added to all the function which don't return anything for the completeness. I'll update the pr.